src/popup.js

```diff
@@ -52,22 +71,22 @@ function request_send_status_updates() {
     }, tabs => {
         if (tabs.length > 0) {
             active_tab_id = tabs[0].id;
-            chrome.tabs.sendMessage(active_tab_id, {what: "send_status_updates"}, checkLastError);
```

Comment:
Can you remove this and other formatting changes? I have no strong feelings about formatting, except that it shouldn't be changed in an unrelated PR.

Reply:
Yes, I will fix that later today (probably).